node/silkworm/common/signal_handler.cpp

```diff
@@ -0,0 +1,130 @@
+/*
+    Copyright 2021 The Silkworm Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include "signal_handler.hpp"
+
+#include <csignal>
+#include <iostream>
+
+namespace silkworm {
+
+uint32_t SignalHandler::sig_count_{0};
```

Comment:
I'd use default initializers in the header, but it's a matter of preference.

Reply:
Right. (I had issues cause I forgot to inline the initializer)

Reply:
Solved vy c265f4f